docs/monitors/jmx.md

```diff
@@ -115,6 +115,37 @@ output.sendDatapoints([
 
 ```
 
+This script collects the same JMX metrics as emitted by default from the [genericjmx monitor](collectd-genericjmx.md).
+It can be used as a starting point for collecting additional JMX metrics.
+
+```groovy
+Map dims(Map args) {
+    // so that dashboard will import
+    d = ["plugin": "GenericJMX"]
+    d.putAll(args ?: [:])
+    d
+}
+dps = [
+    util.makeGauge("gauge.jvm.threads.count", util.queryJMX("java.lang:type=Threading").first().ThreadCount, dims()),
```

Comment:
Let's not encourage people to use the naming conventions of collectd.  Just drop the `gauge.`